_data/compatibility/brd.yaml

```diff
@@ -0,0 +1,89 @@
+      caption: >
+        Sending Transaction - Transaction send default screen. No options for RBF here or in settings menus.
+    - image: /img/compatibility/brd/rbf/send-fee-options-expanded.png
+      caption: >
+        Sending Transaction - Transaction send screen with fee options expanded. No options for RBF. Toggle for fee priority.
+    - image: /img/compatibility/brd/rbf/send-transaction-confirmation.png
+      caption: >
+        Sending Transaction - Transaction send confirmation prompt.
+    - image: /img/compatibility/brd/rbf/transaction-details-sent.png
+      caption: >
+        Attempting Transaction Replacement - No ability to bump transaction fee found. Note Transaction not sent signaling RBF.
+    - image: /img/compatibility/brd/rbf/transaction-list-incoming-rbf.png
+      caption: >
+        Receiving Transaction Signaling RBF - No RBF flag on transaction list screen.
+    - image: /img/compatibility/brd/rbf/transaction-details-incoming-rbf.png
+      caption: >
+        Receiving Transaction Signaling RBF - No RBF flag on transaction details screen.
+    - image: /img/compatibility/brd/rbf/transaction-list-incoming-replacement.png
+      caption: >
+        Receiving Replacement Transaction - On transaction list screen, original transaction shows normal, replacement transaction shows with “Failed” message.
+    - image: /img/compatibility/brd/rbf/transaction-list-incoming-replacement-confirmed.png
+      caption: >
+        Receiving Transaction Signaling RBF - Transaction list screen no longer shows original transaction after the replacement transaction confirms.
+    - image: /img/compatibility/brd/rbf/transaction-details-incoming-replacement.png
+      caption: >
+        Receiving Transaction Signaling RBF - Transaction details of the
+        confirmed replacement transaction. No RBF flag or reference to original
+        transaction.
+segwit:
+  tested:
+    date: "2019-07-23"
+    platforms:
+      - iOS
+    version: "3.14"
+  features:
+    receive:
+      p2sh_wrapped: "false"
+      bech32: "true"
+      default: "p2pkh"
+    send:
+      bech32: "true" 
+      change_bech32: "true"
+      segwit_v1: "Allows sending workflow to complete in the UI. Transaction
+      stays as pending in the transaction list."
+      bech32_p2wsh: "true"
+  examples:
+    - image: /img/compatibility/brd/segwit/receive-screen.png
+      caption: >
+        BRD allows receiving to a bech32 address if you enable segwit in
+        settings.
+    - image: /img/compatibility/brd/segwit/send-screen.png
+      caption: >
+        BRD allows sending to bech32 addresses.
+    - image: /img/compatibility/brd/segwit/settings-enable-segwit.png
+      caption: >
+        BRD has a setting which allows bech32 receive addresses.
+    - image: /img/compatibility/brd/segwit/settings-enable-segwit-details.png
+      caption: >
+        BRD enable segwit details screen.
+    - image: /img/compatibility/brd/segwit/sent-segwit-v1-details.png
```

Comment:
Ditto v1 comment.

Reply:
Ive removed all segwitv1 screenshots from this PR. Going to do the same for other data files.